sprocs/workspace_loads_current.sql

```diff
@@ -39,6 +40,13 @@ BEGIN
     SELECT count(*) INTO workspace_launching_count FROM workspaces AS w WHERE w.state = 'launching';
     SELECT count(*) INTO workspace_relaunching_count FROM workspaces AS w WHERE w.state = 'launching' AND num_nonnulls(w.rebooted_at, w.reset_at) > 0;
     SELECT count(*) INTO workspace_running_count FROM workspaces AS w WHERE w.state = 'running';
+    SELECT count(*) INTO workspace_running_on_healthy_hosts_count
+    FROM
+        workspaces AS w
+        JOIN workspace_hosts AS wh ON (wh.id = w.workspace_host_id)
+    WHERE
+        w.state = 'running'
+        AND wh.state != 'unhealthy';
```

Comment:
I _think_ this is correct. The other option would have been to use `wh.state = 'ready'`, but I'm relatively confident that we want to include workspaces running on `draining` hosts too. There should never be any workspaces running on a `launching`, `terminating`, or `terminated` host, so those shouldn't matter either way.

Reply:
I agree that we want to include workspaces on `draining` hosts, because we'll convert `draining` hosts back to `ready` hosts: https://github.com/PrairieLearn/PrairieLearn/blob/57208cb117c7ecf22bbaa910c2797069454321ca/ee/cron/workspaceHostLoads.js#L151-L152

I think it doesn't matter in practice whether we write `wh.state != 'unhealthy'` or `(wh.state = 'ready' OR wh.state = 'draining')`. I might slightly prefer the latter to make the intent clear, but I could go either way.

Reply:
I'll make that change 👍

Reply:
Changed in 08898ef.